OWNERS_ALIASES

```diff
@@ -13,6 +13,16 @@ aliases:
     - mtnbikenc
     - patrickdillon
     - sdodson
+  libvirt-approvers:
+    - abhinavdahiya
+    - enxebre
+    - praveenkumar
+    - zeenix
+  libvirt-reviewers:
+    - abhinavdahiya
+    - enxebre
```

Comment:
Approvers can also `/lgtm` (in fact, [anyone in the openshift org can `/lgtm`][1]), so I don't think it's worth maintaining a `libvirt-reviewers` alias that is identical to the `libvirt-approvers` alias.  Can we just leave `libvirt-reviewers` empty?  Or am I missing something?

[1]: https://github.com/kubernetes/community/blame/0f0611806cf56e81a99f69d6773271a5d849b3c3/contributors/guide/owners.md#L132-L133

Reply:
I don't think we (CRC team) are part of openshift org) but yeah, you're right. I'll keep `libvirt-reviewers` alias empty for now.